modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,343 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-programming-the-n3000-operator_{context}"]
+= Programming the OpenNESS Operator for Intel FPGA PAC N3000
+
+When the Intel FPGA PAC N3000 is programmed with a vRAN 5G bitstream, the hardware exposes the Intel FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the FEC in the vRAN workload.
+
+As a cluster administrator, you can install the OpenNESS Operator for Intel® FPGA PAC N3000 by using the {product-title} CLI or the web console.
+
+[id="programmimg-n3000-with-a-vran-bitstream_{context}"]
+== Programming the N3000 with a vRAN bitstream
+
+As a cluster administrator, you can program the Intel FPGA PAC N3000 with a vRAN 5G bitstream. This bitstream exposes the Single Root I/O Virtualization (SR-IOV) virtual function (VF) devices used to accelerate the Forward Error Correction (FEC) in the vRAN workload.
+
+[NOTE]
+====
+The role of Forward Error Correction (FEC) is to correct transmission errors, where certain bits in a message can be lost or garbled. Messages can be lost or garbled due to noise in the transmission media, interference, or low signal strength.
```

Comment:
Please remove the note markup.  This is the initial description of what FEC is.  It is not a noteworthy qualification, reminder, or anything else about FEC.

Pleas also lower case "forward error correction" because it is not a proper noun.